From c4b81873d1407c0841738027cc0c8669a0565ba8 Mon Sep 17 00:00:00 2001
From: Binh Nguyen <binh.nguyen.uw@renesas.com>
Date: Tue, 1 Oct 2019 09:18:42 +0700
Subject: [PATCH 160/455] arm64: dts: renesas: r8a774c0-es10: Re-add support
 for RZ/G2E ES1.0

Re-add RZ/G2E ES 1.0, r8a774c0-es10.dtsi includes r8a774c0.dtsi,
and edit device nodes and properties where needed.

Signed-off-by: Binh Nguyen <binh.nguyen.uw@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774c0-es10.dtsi | 12 ++++++++++++
 1 file changed, 12 insertions(+)
 create mode 100644 arch/arm64/boot/dts/renesas/r8a774c0-es10.dtsi

diff --git a/arch/arm64/boot/dts/renesas/r8a774c0-es10.dtsi b/arch/arm64/boot/dts/renesas/r8a774c0-es10.dtsi
new file mode 100644
index 0000000..00576b5
--- /dev/null
+++ b/arch/arm64/boot/dts/renesas/r8a774c0-es10.dtsi
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Device Tree Source for the RZ/G2E (R8A774C0) ES1.0 SoC
+ *
+ * Copyright (C) 2019 Renesas Electronics Corp.
+ */
+
+#include "r8a774c0.dtsi"
+
+&gsx {
+	power-domains = <&sysc R8A774C0_PD_3DG_A>;
+};
-- 
2.7.4

